docs/src/main/resources/content/_using/replication.adoc

```diff
@@ -1,15 +0,0 @@
-:title: Introduction
-:type: using
-:status: published
-:summary: Instructions for performing replication.
-:order: 01
-
-== {title}
-
-Replication is the process of creating a copy of a subset of data and storing it on another
```

Comment:
❓ @clockard Can we move this overview of Replication to the README?

Reply:
Yes. I can add it to the readme